solcast/unmetered_locations.py

```diff
@@ -29,6 +29,16 @@
         "longitude": 78.042142,
         "resource_id": "b926-8fd2-ad3f-e4f5",
     },
+    "Fort Peck": {
+        "latitude": 48.30783,

```

Comment:
5 decimals?

Reply:
Matches the SURFRAD metadata: https://gml.noaa.gov/grad/surfrad/sitepage.html